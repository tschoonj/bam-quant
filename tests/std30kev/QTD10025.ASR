File: C:\xrfdaten\2014KW30\chris\std30kev\std10025.spe
Start: 22.07.2014 09:59:30
Stop: 22.07.2014 10:01:10
BESSY: 298.1666374 298.23439
IONI: 2.01922E-11 2.00771E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98059
Run: 100000
Dead: 2
ICR: 170
OCR: 0
Accumulated BESSY: 59699.0324187
Accumulated Ioni: 4.0338833E-09
N Accumulation: 200
Normfactor:  5.0561413e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mn
exp_x 132.720397949219 132.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  25   1      5.895        -3.         4.        .50
