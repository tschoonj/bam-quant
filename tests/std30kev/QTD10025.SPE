$DATA:
0 2000
23219
14492
6308
1891
352
56
7
0
0
0
0
0
0
0
0
0
2
13
21
15
12
21
12
11
10
7
11
13
15
11
9
10
11
14
9
12
18
12
14
9
17
14
9
13
13
9
12
11
13
8
10
5
11
9
6
5
11
4
12
7
11
12
8
13
4
14
4
6
9
6
6
13
8
8
8
11
11
7
9
5
4
5
12
8
3
7
8
5
5
4
9
10
6
7
9
8
9
7
6
11
10
8
8
7
7
9
10
6
11
9
11
7
9
4
8
8
5
13
6
15
7
8
9
7
3
6
5
5
11
9
10
11
2
9
8
8
7
8
7
6
4
8
7
3
11
6
12
11
11
7
5
11
8
6
3
7
11
5
8
11
9
9
6
8
2
5
7
9
6
12
5
9
7
5
4
12
8
6
5
11
12
4
5
6
6
8
10
3
2
7
5
10
3
8
4
2
2
3
3
3
11
5
3
5
1
6
1
2
4
5
4
4
2
1
4
3
2
2
2
1
1
5
2
2
1
3
0
1
0
1
4
3
2
2
4
4
1
3
3
3
1
4
3
4
3
3
4
0
0
2
3
1
0
0
1
1
1
2
2
4
2
3
1
2
2
1
2
2
1
4
3
2
0
1
1
0
0
3
0
1
0
0
1
1
1
1
1
1
1
2
0
1
1
0
0
0
0
0
2
1
0
0
1
0
0
0
1
1
1
0
0
1
2
3
3
3
2
3
7
6
6
0
2
1
0
1
0
0
0
0
0
0
0
0
2
1
0
0
2
0
0
1
0
1
1
0
2
1
0
0
1
0
2
1
0
1
1
2
1
1
0
0
0
0
0
0
2
0
0
0
2
1
1
1
0
1
1
2
1
1
1
0
0
0
0
2
1
0
1
0
0
1
0
0
0
0
0
0
0
1
0
1
0
0
1
0
1
0
0
1
0
2
1
2
3
3
0
0
0
0
0
0
2
0
1
1
0
0
1
1
1
0
0
0
2
0
1
1
0
0
0
3
2
1
0
3
1
0
1
0
0
1
1
0
1
2
2
0
0
0
3
0
1
0
0
3
1
0
0
0
1
1
1
0
0
0
0
0
0
1
3
1
0
1
0
0
1
0
4
0
1
0
0
0
0
1
0
2
1
1
2
2
3
1
1
1
0
4
0
0
1
2
2
0
1
1
3
1
2
1
2
0
1
1
1
0
1
0
1
2
2
1
0
0
1
0
0
3
2
0
2
1
1
0
1
1
4
4
9
5
15
19
21
19
34
36
28
13
16
18
12
12
4
3
1
7
0
1
3
2
0
0
3
2
1
3
0
0
2
0
3
6
0
1
1
0
0
3
3
1
2
1
1
3
0
2
2
0
1
3
2
3
3
2
1
0
2
3
6
4
6
5
6
9
6
2
3
14
14
10
20
12
9
9
7
7
4
3
5
5
4
5
3
6
1
4
3
2
2
3
3
3
4
1
4
6
0
2
1
6
1
8
1
3
1
2
2
1
2
1
1
5
4
1
2
1
3
3
2
4
3
2
2
4
5
2
5
2
3
2
6
4
4
7
5
2
5
3
2
3
1
5
3
0
1
4
4
4
1
2
2
5
4
0
5
2
3
6
4
5
4
6
3
1
2
3
6
1
3
0
2
6
7
3
5
2
8
2
1
6
5
4
6
0
3
4
9
4
4
3
5
4
6
6
4
5
4
7
6
6
5
10
3
5
1
4
0
6
4
2
6
4
1
3
2
5
11
9
8
13
16
23
34
36
47
64
81
81
92
95
97
83
73
88
76
54
50
27
25
21
10
14
7
8
4
5
4
2
3
5
2
4
6
3
5
2
8
5
2
6
3
5
7
0
2
4
8
1
3
4
4
1
5
1
6
5
8
2
4
3
3
5
6
5
7
1
2
4
2
5
4
10
3
2
4
1
2
11
2
6
3
4
9
3
5
8
1
4
11
7
7
7
6
2
6
4
9
14
4
11
12
16
24
18
8
22
20
11
13
13
11
6
12
6
5
5
5
4
8
9
3
5
6
5
5
9
5
7
3
2
1
3
2
2
2
1
5
2
3
5
2
2
4
4
2
1
1
5
5
2
3
3
2
3
5
5
3
6
3
3
3
6
1
4
2
4
7
3
6
2
2
3
3
3
2
5
2
2
3
1
3
3
6
0
4
7
4
4
3
1
3
4
2
0
4
5
1
2
5
3
1
6
7
2
2
1
2
4
3
3
1
3
2
4
6
1
1
1
4
1
3
2
3
5
4
3
2
3
3
2
1
5
7
1
5
5
2
7
3
4
3
1
3
4
1
2
2
3
6
3
3
6
3
7
4
2
1
3
6
1
0
3
4
4
6
4
2
1
5
3
4
3
2
2
3
3
2
3
2
2
2
1
3
4
2
2
2
2
2
3
3
3
4
1
1
0
2
2
3
4
2
2
4
0
4
3
0
3
3
3
1
0
1
2
4
3
5
5
3
4
2
2
2
3
1
4
2
1
2
5
4
0
4
1
3
7
4
1
5
6
2
1
0
1
2
6
3
5
2
3
3
0
2
6
1
2
3
1
4
1
1
3
3
3
4
5
2
4
4
3
1
2
4
4
4
2
2
3
6
9
4
4
2
2
2
2
4
3
2
2
0
4
2
1
3
0
3
1
2
3
3
8
5
4
1
3
2
2
3
2
1
1
7
1
2
1
5
2
3
3
3
3
2
5
4
1
5
3
2
3
2
3
5
3
0
5
2
0
4
3
3
0
4
0
0
2
3
2
1
2
5
3
2
2
5
0
4
6
3
2
3
1
2
2
4
5
9
4
8
5
14
7
14
10
21
12
15
15
10
11
16
15
8
10
8
11
10
10
6
7
3
3
0
5
2
5
4
2
4
1
2
3
6
2
1
2
1
2
3
2
4
1
2
3
6
2
2
5
3
3
4
2
2
1
4
2
2
2
5
7
4
1
4
3
4
3
7
4
1
3
2
3
1
2
3
5
3
2
3
4
6
4
5
1
1
1
5
2
3
2
3
3
2
4
6
3
2
3
4
4
2
5
3
2
6
5
3
5
3
4
5
2
8
3
2
6
5
5
3
4
6
3
4
3
4
8
2
3
4
4
5
6
4
2
5
8
3
3
5
7
4
4
3
5
4
2
2
5
3
5
6
6
2
5
2
3
6
3
4
9
3
5
4
9
7
6
9
8
4
8
6
3
4
9
12
8
7
4
4
7
4
6
6
9
7
5
7
3
6
11
5
7
3
3
6
4
4
12
5
9
6
6
5
8
5
8
7
7
8
7
5
7
7
5
3
5
1
6
4
10
5
8
5
3
5
5
9
4
10
12
6
6
6
9
7
7
3
7
7
7
7
7
12
6
6
6
10
5
11
7
6
7
7
5
8
9
2
7
11
6
10
14
6
11
10
14
7
9
18
9
6
6
6
8
6
10
10
10
11
12
6
10
4
9
14
14
8
16
16
8
19
11
13
11
12
11
10
19
15
15
12
14
13
16
13
15
9
15
19
13
17
21
22
18
15
15
20
23
11
16
19
15
16
14
23
23
15
10
20
19
31
23
15
25
21
21
30
33
27
18
18
27
24
32
35
26
33
37
30
42
32
40
45
44
23
38
36
35
34
36
33
40
38
39
48
38
39
48
45
42
41
35
55
44
45
47
47
51
38
52
50
59
54
59
33
40
60
61
56
57
69
48
43
49
53
57
52
51
48
50
51
57
51
58
39
39
37
46
34
54
39
44
43
38
43
42
52
39
22
46
37
39
36
31
40
33
47
41
34
46
37
30
34
30
35
34
32
25
26
42
30
28
34
31
32
30
41
38
26
33
28
35
26
16
27
34
32
25
27
26
22
27
25
34
34
26
24
23
25
23
23
18
31
37
22
23
25
20
28
38
35
37
42
37
38
41
27
54
57
52
59
60
66
66
73
77
57
57
62
68
57
74
43
57
53
63
60
59
44
50
53
35
44
45
38
36
25
31
30
28
22
19
13
8
12
7
5
9
10
7
8
4
3
3
2
0
2
1
0
0
0
0
0
1
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10025.spe
Start: 22.07.2014 09:59:30
Stop: 22.07.2014 10:01:10
BESSY: 298.1666374 298.23439
IONI: 2.01922E-11 2.00771E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 98059
Run: 100000
Dead: 2
ICR: 170
OCR: 0
Accumulated BESSY: 59699.0324187
Accumulated Ioni: 4.0338833E-09
N Accumulation: 200
Normfactor:  5.0561413e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mn
 
exp_x 132.720397949219 132.7204
exp_y 150.465789794922 150.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  25   1      5.895        -3.         4.        .50
