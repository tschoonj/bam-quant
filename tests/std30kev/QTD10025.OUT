     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:44  บ   
     บ  Spectrum: QTD10025.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        2 iterations done    บ   
     บ  ChiSquare =     6.8   last change =  -.03%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -49.2 ๑   100.0   บ     -49.2 ๑   103.1   บ   
     บ  GAIN (eV/ch)  บ     21.128 ๑   2.000   บ    21.128 ๑    .542   บ   
     บ det NOISE (eV) บ      225.4 ๑    40.0   บ     225.4 ๑    65.6   บ   
     บ  FANO factor   บ       .048 ๑    .050   บ      .048 ๑    .055   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Mn-Ka                                -3. ๑      4.           บ   
     บ    KA1         5.895   1.00000          -3. ๑      6.       .5  บ   
     บ              281.346    238.40          15.           8.96E-01  บ   
     บ                                                                 บ   
     บ  2 Mn-Kb                                 8. ๑      5.           บ   
     บ    KB1         6.491   1.00000           8. ๑      6.       .9  บ   
     บ              309.555    239.67          13.           9.21E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
